requirements.txt

```diff
@@ -1,4 +1,5 @@
 arrow==1.2.*
+cerberus==1.3.*
```

Comment:
Sergio introduced me to cerberus so it' s been my go-to by default. It doesn't not seem super well maintained at this point. [`jsonschema`](https://github.com/python-jsonschema/jsonschema) might have been a better option? if we were starting a new project with lots of validation i would definitely opt for it.

Reply:
That looks like a nice one. We use cerberus a tiny bit in the Moped API too. Didn't realize there was an alternative. Just starred it ⭐